src/api/modelTypes.js

```diff
@@ -100,6 +100,12 @@ export type User = {|
   bot_type?: number,
   bot_owner?: string,
 
+  // Tim mentions in 2020-02, at
+  // https://github.com/zulip/zulip-mobile/pull/3789#issuecomment-581218576,
+  // that, as of the time of writing, it is always '' for bots, but it may be
+  // omitted later to reduce payload sizes. TODO: So, we should probably
+  // future-proof this field by making it optional, since realm-specific bots
+  // are assigned the User type, with is_bot true.
```

Comment:
nit: TODO should always be at the start of its line of text. (I.e. after the `//`, or the line's leading `*` in the middle of a `/*`-style comment, but before any text.) That helps it stand out.

I think basically I think of it as introducing a new paragraph.